dapp-core/src/test/resources/rmq.properties

```diff
@@ -0,0 +1,8 @@
+rmq.host=rmq
+rmq.irohaExchange=iroha
+rmq.irohaCredential.accountId=dapp1@dapp
```

Comment:
Why 1?

Reply:
just to illustrate the general naming pattern for the instance account